modules/virt-vm-storage-volume-types.adoc

```diff
@@ -27,9 +27,13 @@ recommended method for importing existing virtual machines into
 PVC.
 
 *dataVolume*::
-DataVolumes build on the *persistentVolumeClaim* disk type by managing the process
+DataVolumes build on the `persistentVolumeClaim` disk type by managing the process
 of preparing the virtual machine disk via an import, clone, or upload operation.
 VMs that use this volume type are guaranteed not to start until the volume is ready.
++
+Define DataVolumes with a type of `dataVolume` or `""` (deafult of dataVolume).
```

Comment:
I'd apply the same edits here. Whatever you do, s/deafult/default

Reply:
Changed to edit above. Sorry about the spelling error :-(